public/plugins/testdata/package.json

```diff
@@ -26,6 +26,7 @@
     "@grafana/e2e": "10.1.0-128299pre",
     "@grafana/e2e-selectors": "10.1.0-128299pre",
     "@grafana/eslint-config": "6.0.0",
+    "@grafana/eslint-plugin": "link:../../../packages/grafana-eslint-rules",
```

Comment:
Any reason to use `link:` instead of `workspace:*` protocol here? I'm under the impression this eslint-rules package is a workspace so the latter might be preferable so this reference follows the eslint-rules package dependencies? 🤔

Reply:
I copied the approach from the main `package.json`: https://github.com/grafana/grafana/blob/main/package.json#L107 but I can try defining this as a package, it should work.

Reply:
Ah ok. It's up to you then. I just found it a little odd as generally a workspace would be linked via `workspace` or `portal` rather than `link` but if it's like that elsewhere feel free to ignore this nit.